scripts/rotate-api-key.sh

```
@@ -0,0 +1,104 @@
+
+response=$(curl -sf -X POST "$API_URL/api/admin/keys/rotate" \
+  -H "Content-Type: application/json" \
+  -H "X-API-Key: $CURRENT_KEY" \
+  -d '{"deactivateOld":true}')
+
+new_key=$(python3 - <<'PY'
+import json, sys
+data = json.loads(sys.stdin.read())
+print(data.get("data", {}).get("key", ""))
+PY
+<<< "$response")
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)